resources/views/maps/markers/edit.blade.php

```diff
@@ -60,6 +61,19 @@
                                 </a>
                             </li>
                         </ul>
+                    </div>  
+                    </div>  
+
+                    <div class="pull-left">
+                        <div class="btn-group">
+                                <a role="button" tabindex="-1" class="btn btn-dynamic-delete btn-danger" data-toggle="popover"
+                                    title="{{ __('crud.delete_modal.permanent') }}"
+                                    data-content="
+                                    <a href='#' class='btn btn-danger btn-block' data-toggle='delete-form' data-target='#delete-marker-confirm-form-{{ $model->id}}'>{{ __('crud.remove') }}</a>">
```

Comment:
To keep the pop-out consistent in Kanka, please add this in `data-content` before the button
```php
<p>{{ __(crud.delete_modal.permanent') }}</p>
```

Reply:
Changes implemented